cfme/fixtures/pytest_selenium.py

```diff
@@ -594,8 +594,8 @@ def click(loc, wait_ajax=True, no_custom_handler=False):
     # and then click on current mouse position
     ActionChains(browser()).click().perform()
     # -> using this approach, we don't check if we clicked a specific element
-    if wait_ajax:
-        wait_for_ajax()
+    # if wait_ajax:
```

Comment:
This is temporary . Dev is working on https://bugzilla.redhat.com/show_bug.cgi?id=1470357